libs/core/src/components/pds-icon/docs/pds-icon.mdx

```diff
@@ -1,67 +1,79 @@
-and not creating confusion, we adhere to two principles.
+### When to use
+- Use icons to visually represent actions, features, or common concepts to enhance the user experience.
+- Ideal for buttons, navigation elements, status indicators, or any context where small visual cues can add clarity.
 
-<DocCanvas client:only mdxSource={{
-  react: `<PdsIcon name="thumb-up-filled"></PdsIcon>`,
-  webComponent: `<pds-icon name="thumb-up-filled"></pds-icon>`
-}}>
-  <>
-    <pds-icon name="thumb-up-filled"></pds-icon>
-  </>
-</DocCanvas>
+### When not to use
+- Avoid using icons as the sole means of communication without accompanying text or labels, as this may confuse users.
+- Do not use icons for complex ideas that require explanation or context that cannot be clearly conveyed through visuals alone.
+
+### Accessibility
+- Ensure sufficient contrast between the icon and the background for visibility.
+- Don't rely solely on icons for critical actions.
 
 ## Properties
 
-<DocArgsTable componentName='pds-icon' docSource={pdsDocsJson} />
+<DocArgsTable componentName="pds-icon" docSource={pdsDocsJson} />
 
-### color
-The `color` property allows the icon to render in the specified color. This can be either string or hex format.
+### Color
+
+The `color` property allows the icon to be displayed in a specified color. Accepted formats include:
+
+- String: `red`
+- HEX:  `#FF0000`
+- RGB:  `rgb(255, 0, 0)`
+- HSL: `hsl(0, 100%, 50%)`
+- Pine CSS variables: `var(--pine-color-red-500)`
+
+This allows for flexibility with any CSS-supported color format.
 
 <DocCanvas client:only mdxSource={{
   react: `
-<PdsIcon name="bulb" size="small"></PdsIcon>
-<PdsIcon name="bulb" size="regular"></PdsIcon>
-<PdsIcon name="bulb" size="medium"></PdsIcon>
-<PdsIcon name="bulb" size="large"></PdsIcon>
+<PdsIcon name="bulb" color="red"></PdsIcon>
+<PdsIcon name="bulb" color="orange"></PdsIcon>
+<PdsIcon name="bulb" color="green"></PdsIcon>
+<PdsIcon name="bulb" color="blue"></PdsIcon>
 `,
   webComponent: `
-<pds-icon name="bulb" size="small"></pds-icon>
-<pds-icon name="bulb" size="regular"></pds-icon>
-<pds-icon name="bulb" size="medium"></pds-icon>
-<pds-icon name="bulb" size="large"></pds-icon>`
+<pds-icon name="bulb" color="red"></pds-icon>
+<pds-icon name="bulb" color="orange"></pds-icon>
+<pds-icon name="bulb" color="green"></pds-icon>
+<pds-icon name="bulb" color="blue"</pds-icon>`
 }}>
   <>
-    <pds-icon name="bulb" size="small"></pds-icon>
-    <pds-icon name="bulb" size="regular"></pds-icon>
-    <pds-icon name="bulb" size="medium"></pds-icon>
-    <pds-icon name="bulb" size="large"></pds-icon>
+    <pds-icon name="bulb" color="red"></pds-icon>
+    <pds-icon name="bulb" color="orange"></pds-icon>
+    <pds-icon name="bulb" color="green"></pds-icon>
+    <pds-icon name="bulb" color="blue"></pds-icon>
   </>
 </DocCanvas>
 
-It also accepts any custom size attribute.
+### Size
+
+The `size` property also accepts custom sizes.
```

Comment:
We should probably include what t-shirt sizes are available.

below is what was taken from code. 
```
| 'small'   // 12px
| 'regular'  // 16px
| 'medium'  // 20px
| 'large'   // 24px
```

Reply:
Added. Good call.